packages/svelte/messages/compile-errors/template.md

```diff
@@ -192,6 +192,14 @@
 
 > %thing% is invalid inside <%parent%>
 
+HTML has some restrictions where certain elements can appear. For example, a `<div>` inside a `<p>` is invalid. Some violations "only" result in invalid HTML, others result in the HTML being repaired by the browser, resulting in content shifting around. Some examples:
+
+- `<p>hello <div>world</div></p>` will result in `<p>hello </p><div>world</div><p></p>` for example (the `<div>` autoclosed the `<p>`)
+- `<option><div>option a</div></select>` will result in `<option>option a</option>` (the `<div>` is removed)
```

Comment:
The `</select>` closing tag should probably be `</option>` here.

Reply:
You're right, thanks for noticing!